sphinx/writers/latex.py

```diff
@@ -361,6 +387,67 @@ def get_colspec(self):
         else:
             return '{|' + ('l|' * self.colcount) + '}\n'
 
+    def add_cell(self, height, width):
+        """Adds a new cell to a table.
+
+        It will be located at current position: (``self.row``, ``self.col``).
+        """
+        self.cell_id += 1
+        for col in range(width):
+            for row in range(height):
+                assert self.cells[(self.row + row, self.col + col)] == 0
+                self.cells[(self.row + row, self.col + col)] = self.cell_id
```

Comment:
@tk0miya (my Python teacher ;-) ) my earlier comment was based on a wrong understanding of   `assert` keyword...